source/hpack/decoder.d

```diff
@@ -0,0 +1,257 @@
+module HPACK.decoder;
```

Comment:
Ah, one thing I've missed: this should be all-lowercase to not produce problems due to the differing case of the folder name when the library is imported as an external dependency (and also to be idiomatic style).

Reply:
Thank you for reminding me, I left them upper case to remember to change them in case we wanted to integrate it directly with `vibe-http` instead of using it as external dependency.